### DISCUSSION TITLE
posts/algorithm-plan/

### DISCUSSION THREAD

**Discussion by giscus** [ddiyooong/ddiyooong.github.io#3] [General]
# posts/algorithm-plan/

백준 온라인 강의 알고리즘 기초1 , 알고리즘 기초2 에 포함되어 있는 문제를 가져옴

https://ddiyooong.github.io/posts/algorithm-plan/

**Comment by Ji-soo708** [comment]
저는 진정한 개발자라면 이펙티브 자바를 먼저 공부하는 게 맞다고 생각합니다.

**Reply by ddiyooong** [reply]
널세이프한 코틀린의 세계로 갑시다.

**Reply by Ji-soo708** [reply]
@kth990303 이 의견에 어떻게 생각하시나요.

**Reply by ddiyooong** [reply]
### 수정안댐?

**Reply by Ji-soo708** [reply]
discussion까지 와야하다니 👀

**Reply by ddiyooong** [reply]
오 보니까 댓글 닉네임 옆에 1 minute ago 클릭하면 디스커션으로 바로 가네요

**Reply by kth990303** [reply]
@Ji-soo708 반쯤 동의합니다ㅋㅋ
